src/selectors/petitionTeaser.js

```diff
@@ -16,14 +16,14 @@ export default (petition) => {
     title: petition.title,
     footer: {
       info: {
-        city: petition.city || 'Aargau',
+        city: petition.city,
         owner: getPetitionAuthor(petition.owner || {})
       },
-      dialTime: {
+      timeMetric: {
         figure: daysRemaining,
-        percentage: calculatePercentage(daysRemaining, settings.daysToVote)
+        percentage: 100 - calculatePercentage(daysRemaining, settings.daysToVote)
```

Comment:
I think it is ok for now as it is. If we need that logic more often we can bake it in